<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartCroppingOfBagsClass</class>
 <widget class="QMainWindow" name="SmartCroppingOfBagsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1087</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCroppingOfBags</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="9,1">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lb_show">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lb_Ngshow">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>611</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>611</width>
           <height>161</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 18px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
         </property>
         <property name="title">
          <string>连接状态</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>591</width>
            <height>121</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>相机连接状态:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_xiangjilianjiezhuangtai">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>连接失败</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>板卡连接状态:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_bankalianjiezhuangtai">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>连接失败</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>230</y>
           <width>611</width>
           <height>81</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,1">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pingbiquyu">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>屏蔽区域</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>320</y>
           <width>611</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,2,1">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="ckb_zhinengcaiqie">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox {
    /* 默认样式 */
    spacing: 10px; /* 文字和框框之间的间距 */
    font-size: 14px; /* 字体大小 */
}

QCheckBox::indicator {
    /* 按钮的大小 */
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    /* 未选中状态的样式 */
    background-color: lightgray; /* 背景颜色 */
    border: 1px solid darkgray; /* 边框颜色 */
}

QCheckBox::indicator:unchecked:hover {
    /* 未选中状态鼠标悬停时的样式 */
    background-color: gray; /* 背景颜色 */
}

QCheckBox::indicator:checked {
    /* 选中状态的样式 */
    background-color: rgb(255, 152, 83); /* 背景颜色 */
    border: 1px solid darkred; /* 边框颜色 */
}

QCheckBox::indicator:checked:hover {
    /* 选中状态鼠标悬停时的样式 */
    background-color: darkred; /* 背景颜色 */
}</string>
            </property>
            <property name="text">
             <string>智能裁切</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_chanliangqingling">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>产量清零</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>370</y>
           <width>611</width>
           <height>241</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 18px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
         </property>
         <property name="title">
          <string>生产状态</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>32</y>
            <width>591</width>
            <height>201</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>生产总量:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lb_shengchanzongliang">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Value</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>良率:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lb_lianglv">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Value</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>废品数量:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lb_feipinshuliang">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Value</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>平均袋长:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lb_pingjundaichang">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Value</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>620</y>
           <width>611</width>
           <height>81</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="0,0,0">
          <item>
           <widget class="QCheckBox" name="ckb_tifei">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox {
    /* 默认样式 */
    spacing: 10px; /* 文字和框框之间的间距 */
    font-size: 14px; /* 字体大小 */
}

QCheckBox::indicator {
    /* 按钮的大小 */
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    /* 未选中状态的样式 */
    background-color: lightgray; /* 背景颜色 */
    border: 1px solid darkgray; /* 边框颜色 */
}

QCheckBox::indicator:unchecked:hover {
    /* 未选中状态鼠标悬停时的样式 */
    background-color: gray; /* 背景颜色 */
}

QCheckBox::indicator:checked {
    /* 选中状态的样式 */
    background-color: rgb(255, 152, 83); /* 背景颜色 */
    border: 1px solid darkred; /* 边框颜色 */
}

QCheckBox::indicator:checked:hover {
    /* 选中状态鼠标悬停时的样式 */
    background-color: darkred; /* 背景颜色 */
}</string>
            </property>
            <property name="text">
             <string>剔废功能</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckb_huikan">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox {
    /* 默认样式 */
    spacing: 10px; /* 文字和框框之间的间距 */
    font-size: 14px; /* 字体大小 */
}

QCheckBox::indicator {
    /* 按钮的大小 */
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    /* 未选中状态的样式 */
    background-color: lightgray; /* 背景颜色 */
    border: 1px solid darkgray; /* 边框颜色 */
}

QCheckBox::indicator:unchecked:hover {
    /* 未选中状态鼠标悬停时的样式 */
    background-color: gray; /* 背景颜色 */
}

QCheckBox::indicator:checked {
    /* 选中状态的样式 */
    background-color: rgb(255, 152, 83); /* 背景颜色 */
    border: 1px solid darkred; /* 边框颜色 */
}

QCheckBox::indicator:checked:hover {
    /* 选中状态鼠标悬停时的样式 */
    background-color: darkred; /* 背景颜色 */
}</string>
            </property>
            <property name="text">
             <string>回看功能</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckb_yinshuazhiliangjiance">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox {
    /* 默认样式 */
    spacing: 10px; /* 文字和框框之间的间距 */
    font-size: 14px; /* 字体大小 */
}

QCheckBox::indicator {
    /* 按钮的大小 */
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    /* 未选中状态的样式 */
    background-color: lightgray; /* 背景颜色 */
    border: 1px solid darkgray; /* 边框颜色 */
}

QCheckBox::indicator:unchecked:hover {
    /* 未选中状态鼠标悬停时的样式 */
    background-color: gray; /* 背景颜色 */
}

QCheckBox::indicator:checked {
    /* 选中状态的样式 */
    background-color: rgb(255, 152, 83); /* 背景颜色 */
    border: 1px solid darkred; /* 边框颜色 */
}

QCheckBox::indicator:checked:hover {
    /* 选中状态鼠标悬停时的样式 */
    background-color: darkred; /* 背景颜色 */
}</string>
            </property>
            <property name="text">
             <string>印刷质量检测</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>710</y>
           <width>611</width>
           <height>151</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 18px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
         </property>
         <property name="title">
          <string>亮度检测</string>
         </property>
         <widget class="QWidget" name="layoutWidget_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>581</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="0,0,0,0,0">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_down">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_baoguang">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>btn</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_up">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>581</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="3,1,2">
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="lb_liangduzhuangtai">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>亮度正常</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lb_liangdu">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_daizizhonglei">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>白色袋</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>870</y>
           <width>611</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="3,1">
          <item>
           <widget class="QPushButton" name="btn_normalParam">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>参数设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_setParam">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
